lmnet/lmnet/networks/object_detection/lm_fyolo.py

```diff
@@ -167,74 +168,38 @@ def __init__(
             activation_quantize_kwargs(dict): Initialize kwargs for activation quantizer.
         """
 
-        super().__init__(
+        LMFYolo.__init__(
+            self,
             *args,
             **kwargs,
         )
 
-        self.quantize_first_convolution = quantize_first_convolution
-        self.quantize_last_convolution = quantize_last_convolution
-
-        activation_quantizer_kwargs = activation_quantizer_kwargs if not None else {}
-        weight_quantizer_kwargs = weight_quantizer_kwargs if not None else {}
-
+        BaseQuantize.__init__(
+            self,
+            activation_quantizer,
+            activation_quantizer_kwargs,
+            weight_quantizer,
+            weight_quantizer_kwargs,
+            quantize_first_convolution,
+            quantize_last_convolution,
+        )
         assert callable(weight_quantizer)
```

Comment:
Why  `assert callable(weight_quantizer)` and `assert callable(activation_quantizer)` remains in here?

Reply:
Because the original version, some network classes call `assert callable(weight_quantizer)` and `assert callable(activation_quantizer)`, while some classes do not call,
I put these `assert` only in the classes of which original one calls the `assert`.

Reply:
Should I include 
`assert callable(weight_quantizer)` and `assert callable(activation_quantizer)` 
into the base classes and ignore 
`assert weight_quantizer` and `assert activation_quantizer`?

Reply:
I changed the `assert ..` to `assert callable( .. )` in `BaseQuantize` class and remove all `assert callable ( .. )` from all subclasses.